src/neptune/handler.py

```diff
@@ -736,6 +736,20 @@ def __delitem__(self, path) -> None:
     @feature_temporarily_unavailable
     @check_protected_paths
     def pop(self, path: str = None, *, wait: bool = False) -> None:
+        """Completely removes the namespace and all associated metadata stored under the path.
+
+        Args:
+            path: Path of the namespace to be removed.
+            wait: By default, logged metadata is sent to the server in the background.
+                With this option set to `True`, Neptune first sends all data before executing the call.
```

Comment:
```suggestion
                With this option set to `True`, Neptune sends all data before executing the call.
```
Avoids redundancy. Or `..., Neptune first sends all data, then executes the call`.

Reply:
This parameter is used in numerous places, so we have a lot of instances to update :)